content/en/docs/instrumentation/go/manual.md

```diff
@@ -720,6 +720,48 @@ func init() {
 }
 ```
 
+### Registering Views
+
+A View provides SDK users with the flexibility to customize the metrics that are
+output by the SDK. They can customize which metric instruments are to be
+processed or ignored. They can customize aggregation and what attributes are to
+be reported on metrics.
+
+Every instrument has a default view, which retains the original name,
+description, and attributes, and has a default aggregation that is based on the
+type of instrument. When a registered view matches an instrument, the default
+view is replaced by the registered view. Additional registered views that match
+the instrument are additive, and result in multiple exported metrics for the
+instrument.
```

Comment:
This is copy pasted from https://opentelemetry.io/docs/instrumentation/java/manual/#views